Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1314,6 +1316,20 @@ void WebProcessProxy::processDidTerminateOrFailedToLaunch(ProcessTerminationReas
         remotePage->processDidTerminate(coreProcessIdentifier());
 }
 
+AutoGeneratedTranscriptionManager& WebProcessProxy::autoGeneratedTranscriptionManager()
+{
+    ensureAutoGeneratedTranscriptionManager();
+    return *m_autoGeneratedTranscriptionManager;
+}
+
+void WebProcessProxy::ensureAutoGeneratedTranscriptionManager()
+{
+    if (!m_autoGeneratedTranscriptionManager) {
+        m_autoGeneratedTranscriptionManager = WTF::makeUnique<AutoGeneratedTranscriptionManager>(&protectedConnection().get());
+        addMessageReceiver(Messages::AutoGeneratedTranscriptionManager::messageReceiverName(), *m_autoGeneratedTranscriptionManager);
+    }
+}
```

Comment:
Unless `ensureAutoGeneratedTranscriptionManager` will be used in other methods, you might as well combine these.